packages/notifi-axios-utils/lib/utils/axiosRequest.ts

```diff
@@ -15,7 +15,7 @@ const makeRequestInternal = async <Input, Result>(
   config?: AxiosRequestConfig<Input>,
 ): Promise<Result> => {
   const { data } = await axiosInstance.post<PostResponse<Result>>(
-    '/',
+    `/${encodeURIComponent(resultKey)}`,
```

Comment:
Is this essentially the same as setting x-request-id header? Can we do that instead, or too? We can send that back down in the header as x-response-id

Reply:
This is just to make it easier to identify GQL requests when troubleshooting:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/105243600/176381666-1d4796de-a905-4872-9d96-782475b11bbd.png">

I opened [MVP-822](https://notifi.atlassian.net/browse/MVP-822) and [MVP-822](https://notifi.atlassian.net/browse/MVP-823) to track the work required to support an `x-request-id` header